Sources/LocheckLogic/Validators/parseAndValidateXCStrings.swift

```diff
@@ -12,42 +12,32 @@ import Foundation
  Directly compare two `.strings` files
  */
 public func parseAndValidateXCStrings(
-    base: File,
-    translation: File,
+    base: [File],
+    translation: [File],
     translationLanguageName: String,
     problemReporter: ProblemReporter) {
-    problemReporter.logInfo("Validating \(translation.path) against \(base.path)")
-
-    let baseStrings = base.lo_lines.enumerated()
-        .compactMap {
-            LocalizedStringPair(
-                string: $0.1,
-                path: base.path,
-                line: $0.0 + 1)
+    let collectLines = { (files: [File], baseStringMap: [String: FormatString]) -> [LocalizedStringPair] in
+        files.flatMap { file in
+            file.lo_getLines(problemReporter: problemReporter)?
+                .enumerated()
+                .compactMap {
+                    LocalizedStringPair(
+                        string: $0.1,
+                        path: file.path,
+                        line: $0.0 + 1)
+                } ?? []
```

Comment:
Note that even though strings are combined into a single list per language, each string is still associated with its original file.